crates/esthri-cli/src/http_server.rs

```diff
@@ -357,43 +362,14 @@ async fn create_error_monitor_stream<T: Stream<Item = io::Result<BytesMut>> + Un
     }
 }
 
-// Work around for https://github.com/Majored/rs-async-zip/issues/11 By default,
```

Comment:
I believe this was fixed upstream, there is a new release and @samvrlewis's bug was marked fixed: https://github.com/Majored/rs-async-zip/issues/11